app/views/users/show.html.erb

```diff
@@ -0,0 +1,37 @@
+<h1>All events created by you</h1>
+
+<h2>Previously Created Events</h2>
+<ul> 
+  <% @past.each do | event | %>
+    <li>
+    <%= link_to event.name, event_path(event) %>
+    </li>
+  <% end %>
+</ul>
+
+<h2>Upcoming Created Events</h2>
+<ul> 
+  <% @future.each do | event | %>
+  <li>
+  <%= link_to event.name, event_path(event) %>
+  </li>
+  <% end %>
+</ul>  
+
+<h1>Events you are attending</h1>
+
+<h2>Past Events</h2>
+<% current_user.attended_events.past.each do |event| %>
+  <strong><%= event.name%> created by: </strong>
+  <p><%= event.creator.name %></p>
+  <p><%= event.date %></p>
+  <p><%= link_to "Description", event_path(event.id) %></p>
+<% end %>
+
+<h2>Future Events</h2>
+<% current_user.attended_events.future.each do |event| %>
+  <strong><%= event.name %> created by: </strong>
+  <p><%= event.creator.name %></p>
+  <p><%= event.date %></p>
+  <p><%= link_to "Description", event_path(event.id) %></p>
+<% end %>
```

Comment:
`current_user.attended_events.past` is not good in the views. Created views variables in the controller to easy the reading and simplified your code
That is, for example
```
@past_events = current_user.attended_events.past
```